src/automations/checkInactivity.js

```diff
@@ -98,25 +99,45 @@ async function checkInactivePullRequest(pullRequest) {
   }
 }
 
+async function extractPRLabels(repoOwner, repoName, number) {
+  const labels = await this.issues.getIssueLabels({
+    owner: repoOwner, repo: repoName, number: number
+  });
+  const inactiveLabel = labels.data.find(l => {
+    return l.name === this.cfg.activity.inactive;
+  });
+  const reviewedLabel = labels.data.find(l => {
+    return l.name === this.cfg.activity.pullRequests.reviewed.label;
+  });
+
```

Comment:
You should also be able to return the `needsReviewLabel` instead of just relying on the presence of the reviewed label.

Reply:
In that case, should I just access the `needsReviewLabel` instead? This issue demands that an issue should not be marked inactive if the 'needs review' label is present, rather than the absence of the 'reviewed' label.

Reply:
Yeah, that's what I meant.

Reply:
Alright, so I have accessed both the `needsReviewLabel` and `reviewedLabel`, reason being that a couple of if blocks make use of the absence/presence of `reviewedLabel` in `checkInactivePullRequest`